ajax/libs/watch/package.json

```diff
@@ -1,6 +1,14 @@
 {
   "name": "watch",
-  "filename": "jquery.watch.min.js",
+  "filename": "watch.min.js",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/darcyclarke/Watch.js.git",
+    "basePath": "",
+    "files": [
+      "watch.js"
```

Comment:
Please use `"*watch*.+(js|map)"` here, so that it can works with old versions, e.g. https://github.com/darcyclarke/Watch.js/tree/v2.0.1